doctests/dt-geo.js

```diff
@@ -0,0 +1,59 @@
+// EXAMPLE: geo_tutorial
+// HIDE_START
+import assert from 'assert';
+import { createClient } from 'redis';
+
+const client = createClient();
+await client.connect();
+// HIDE_END
+
+// REMOVE_START
+await client.del('bikes:rentable')
+// REMOVE_END
+
+// STEP_START geoAdd
+const res1 = await client.geoAdd('bikes:rentable', { 
```

Comment:
I know this is nit-picky, but can we put a space after each of the object properties here. So, like this:
```javascript
const res2 = await client.geoAdd('bikes:rentable', {
  longitude: -122.2674626,
  latitude: 37.8062344,
  member: 'station:2'
});
```

Reply:
Done!